src/main/scala/org/scalawag/jibe/Main.scala

```diff
@@ -63,7 +63,7 @@ object Main {
       val resultsDir = new File("results") / date
       val results = Executive.apply(orderedMandate, ssh, UbuntuCommander, resultsDir / "raw")
 
-      Reporter.generate(resultsDir / "raw", resultsDir / "html" / "index.html")
+      Reporter.generate(resultsDir / "raw", resultsDir / "html" / "index.html", resultsDir.getParentFile / "latest.html")
```

Comment:
My vote is for "another option", i.e. Reporter should be in charge of the whole results directory and managing what is produced, linked, etc., there.